src/templates/parameter_file_templates/run_time.par

```diff
@@ -3,8 +3,9 @@ project_root=<path to project root>
 rollup_directory=<path to output directory for election result roll-ups>
 results_file=<path to raw results file from election agency>
 juris_name=<name of jurisdiction's directory>
-db_paramfile=<parameter file for connecting to database>
+db_paramfile=<path to parameter file for connecting to database>
 munger_name=<name of munger's directory>
 db_name=<name of database>
 top_reporting_unit=<internal db name for the overall jurisdiction>
 sub_reporting_unit_type=<geography to roll results up to>
+atomic_ru_type=<smallest ReportingUnit type >
```

Comment:
@Raghu-Srungavarapu Why do we need this?

Reply:
We don't need it anymore, since we're picking vote counts by datafile, not by ReportingUnitType as we did previously.

Reply:
@Raghu-Srungavarapu , please delete this line so we can merge this pull request.